pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
     "python-dotenv>=1.0.1,<1.1.0",
     "rich>=12.4.4,<13.0.0",
     # If adding support for a new major/minor Flask version below, remember to add a tox env to matrix test it.
-    "Flask>=2.1.1,<=3.0.3",
+    "Flask>=2.1.1,!=2.3.0",
```

Comment:
They are running Flask 3 on this repo, not sure if this will break for them? https://github.com/communitiesuk/funding-service-design-fund-application-builder/blob/abbdfa61a14f47ae5423709af8c5a3787a16c6a9/requirements.in#L31

Reply:
That is fine - this doesn't exclude Flask 3 - it just excludes specifically Flask 2.3.0, [which had a seemingly unexpected breaking change that was reverted in 2.3.1](https://flask.palletsprojects.com/en/2.3.x/changes/#version-2-3-1).

This PR has a whole set of tox test runs that run against Flask 3, so it definitely doesn't remove compatibility.